linode/manager
# M3-5597: Storybook updates for Analytics & Graphs

## Description

Updates Storybook stories for Analytics & Graphs 📈

- [x] Convert Gauge to MDX
- [x] Convert Line Graph to MDX
- [x] Remove `Longview Line Graph` example
- [x] Update Gauge percent story and add copy
- [x] Update Line Graph and add copy
- [x] Add Network transfer graph story

## How to test

- `yarn storybook`
- Check all stories under `UI Elements` ➡️ `Analytics & Graphs`
- Check each story for style, spelling, usability of args tables, etc...
